src/iocore/cache/unit_tests/test_CacheDir.cc

```diff
@@ -88,7 +88,7 @@ class CacheDirTest : public CacheInit
       return EVENT_DONE;
     }
 
-    vol_dir_clear(vol);
+    vol->clear_dir();
```

Comment:
Do we plan to rename this to `stripe` instead of `vol` later?

Reply:
Yes. Renaming local variable will be in another PR. I sliced changes into multiple PRs, because it becomes over 1K lines changes.